app/src/main/java/org/oppia/android/app/utility/FontScaleConfigurationUtil.kt

```diff
@@ -18,19 +19,21 @@ class FontScaleConfigurationUtil @Inject constructor() {
     configuration.fontScale = getReadingTextSizeConfigurationUtil(readingTextSize)
     val metrics: DisplayMetrics = context.resources.displayMetrics
     val windowManager = context.getSystemService(WINDOW_SERVICE) as? WindowManager
-    // TODO(#3616): Migrate to the proper SDK 30+ APIs.
+    // TODO (#3616): Migrate to the proper SDK 30+ APIs.
```

Comment:
```suggestion
    // TODO(#3616): Migrate to the proper SDK 30+ APIs.
```

remove that space to fix the failing static check